pwndbg/gdblib/__init__.py

```
@@ -17,7 +17,7 @@
 __all__ = ["ctypes", "memory", "typeinfo"]
 
 # Export parsed GDB version
-gdb_version = tuple(map(int, re.search(r"(\d+)[^\d]+(\d+)", gdb.VERSION).groups()))
+gdb_version = tuple(map(int, re.search(r"(\d+)[^\d]+(\d+)", gdb.VERSION).groups()))  # type: ignore[attr-defined]
```

@gsingh93 

Using # type: ignore[attr-defined] here to ignore the error (gdb.VERSION), works locally - the CI/CD testing check fails on it, though

---

Nevermind - saw https://github.com/pwndbg/pwndbg/pull/2064. Re-installed dependencies, works now.